### ISSUE TITLE
Is there a theoretical range of values for the COMET regressor? 

### ISSUE THREAD

**Issue by alvations** [Unbabel/COMET#52]
Is there a theoretical range of values for the COMET regressor? 

Since the final estimator layer is an FFN https://github.com/Unbabel/COMET/blob/master/comet/models/regression/regression_metric.py#L95

Which goes to https://github.com/Unbabel/COMET/blob/85b0c8ff24fb63273283807cd892d3ef44b89a3a/comet/modules/feedforward.py#L58 

Is the theoretical range `(-inf, inf)`, https://pytorch.org/docs/1.9.1/generated/torch.nn.Linear.html?

Are there some table of practical ranges that the comet owners/contributors/users have found for varying languages and length?

**Comment by ricardorei** [comment]
Hi @alvations 

Yes, the scores are unbounded for most COMET models (except `wmt20-comet-da-qe` that has a sigmoid in the end). Nonetheless, since they are trained to regress on a [z-scores](https://www.statology.org/z-score-normalization/) they usually range somewhere between -2.5 to 1.2. You can observe that in [this plot](https://unbabel.github.io/COMET/html/models.html#faq):

![wmt20_scores_distribution](https://user-images.githubusercontent.com/17256847/149377413-46798cec-6a03-4588-b8ea-8bbcf0a8ae37.png)

That this plot from WMT20 en-de data which is a high resource and high quality language pair. You can expect this distribution to have a smaller mean for low resource languages.

The max value and min value should be more or less the same because the model does not "risk" much outside those values.

**Comment by alvations** [comment]
Thanks for the explanation! It'll make sense if we somehow distill the model and put a softmax at the final layer so that it spits out `[0, 1.0]`.
